pipelines/batch/src/main/java/org/openmrs/analytics/JdbcFetchUtil.java

```diff
@@ -13,20 +13,24 @@
 // limitations under the License.
 package org.openmrs.analytics;
 
+import java.sql.Blob;
```

Comment:
How much of the functionality in this file is shared between OpenMRS and HAPI? If not much, then maybe we should separate this file into two, e.g., `JdbcFetchOpenMrs.java` and `JdbcFetchHapi.java`. Another option is to add the distinction in function names, e.g., `FetchOpenMrsUuids`, `FetchHapiResources`, etc. WDYT?